DTypes 01 – Strings/DTYPES01-019.js

```diff
@@ -0,0 +1,22 @@
+/*
+DTYPES01-019: The following permanent text is given:
+Primitive
+Print on the console the following:
+On the first line – print the first character.
+On the second line – print first two characters.
+On the third line – print first three characters and so on until the whole word is 
+printed.
+Pad each line with space character(s), so the text is right justified.
+*/
+
+const Primitive = 'Primitive';
```

Comment:
It's a good practice to use capital letters only when declairing const variables. The solution is correct.